_IBC/archway-bitcanna.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "archway",
+    "client_id": "07-tendermint-35",
```

Comment:
client_id: 07-tendermint-35 trusting period is not 2/3 of the unbodning period

>         trusting_period: 604800s,
>         unbonding_period: 1209600s,

Reply:
trusting period must be: 806400s to comply with requirements [here](https://github.com/archway-network/networks#requirements-for-the-ibc-channel)

Reply:
> client_id: 07-tendermint-35 trusting period is not 2/3 of the unbodning period

I am trying to figure out how these values ended up here.

> trusting period must be: 806400s to comply with requirements [here](https://github.com/archway-network/networks#requirements-for-the-ibc-channel)

As per the referenced link, "trusting period must be 1209600s".

---

Why this is confusing to me is Hermes should pick up unbonding period auto-magically and the trusting period was specified by me as `14days` which is the expected `1209600s`, so the client should have been generated as:
> ```text
> trusting_period: 1209600s,
> unbonding_period: 1814400s,
> ```

I will try to figure out why it ended up the way it did, and update this PR with new client/connection/channel.

Reply:
Got it - it's a misconfiguration for our BitCanna side.